core/templates/pages/release-coordinator-page/features-tab/features-tab.component.html

```diff
@@ -1,5 +1,5 @@
 <oppia-loading-message [message]="loadingMessage"></oppia-loading-message>
-<div *ngFor="let featureFlag of featureFlags; index as index">
+<div *ngFor="let featureFlag of featureFlagViewModels; index as index">
```

Comment:
I'm not sure I understand why we're introducing a naming inconsistency here. Normally it's "for {{singular}} of {{plural}}"; why are we changing singular and plural to different things?

Reply:
Right, I have updated this.